src/ext/ddtrace.c

```diff
@@ -70,7 +69,6 @@ static PHP_MINIT_FUNCTION(ddtrace) {
     ddtrace_initialize_config(TSRMLS_C);
 
     ddtrace_install_backtrace_handler();
-    ddtrace_dispatch_init(TSRMLS_C);
```

Comment:
😱 Very nice catch! ❤️

Reply:
Found it while debugging why phpdbg fails surprised me it haven't failed sooner. Or maybe it did 😨